docs/user/01-manager.md

```diff
@@ -1,15 +1,18 @@
 # Telemetry Manager
 
+As the core element of the Telemetry module, Telemetry Manager manages the lifecycle of other Telemetry module components by watching user-created resources.
+
 ## Module Lifecycle
 
-Kyma's Telemetry module ships Telemetry Manager as its core component. Telemetry Manager is a Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) that is described by a custom resource of type Telemetry. Telemetry Manager implements the Kubernetes controller pattern and manages the whole lifecycle of all other components covered in the Telemetry module.
-Telemetry Manager watches for the user-created Kubernetes resources: LogPipeline, TracePipeline, and MetricPipeline. In these resources, you specify what data of a signal type to collect and where to ship it.
-If Telemetry Manager detects a configuration, it rolls out the relevant components on demand.
+The Telemetry module includes Telemetry Manager, a Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) that's described by a custom resource of type Telemetry. Telemetry Manager has the following tasks:
+
+- Watch for the user-created Kubernetes resources LogPipeline, TracePipeline, and MetricPipeline. In these resources, you specify what data of a signal type to collect and where to ship it.
+- If it finds such a custom resource: Roll out the relevant components on demand and keep it in sync with the pipeline.
 
 ![Manager](assets/manager-lifecycle.drawio.svg)
 
 ## Module Configuration
```

Comment:
```suggestion
## Module Configuration and Status

For available configuration options and the overall status of the module, please see the specification of the related [Telemetry resource](./resources/01-telemetry.md).
```
Let's condense the two sections into one

Reply:
Then enrich the example in the telemetry resource .md with a spec like:
```
spec:
  trace:
    gateway:
      scaling:
        type: Static
        static:
          replicas: 3
  metric:
    gateway:
      scaling:
        type: Static
        static:
          replicas: 4
```

Reply:
Please confirm: 
Do you want to add the code sample to the document in DITA?
And/or do you want me to edit the code sample in `01-manager.md`? This is what it looks like now:

<img width="918" alt="image" src="https://github.com/kyma-project/telemetry-manager/assets/76950046/c64640ec-41a8-4987-837f-55df775a325e">

And/or do you want me to edit the `resources/01-telemetry.md`? And in that case, drop the code sample from `01-manager.md` and not add it to DITA?

Reply:
I want to have the additional spec listed in the existing sample in resources/01-telemetry.md, so that in the manager docu there is no sample anymore. And it should be in both sources the same.

Reply:
done with [9036728](https://github.com/kyma-project/telemetry-manager/pull/1247/commits/90367282dbf6da102f1b4df72214d0be3381b5e9)